docs/getting-started/products/ylockers/overview.md

```diff
@@ -1,8 +1,111 @@
+
+:::
+
+yLockers are a category of assets built by Yearn designed to tokenize locked governance positions in external DeFi protocols. This asset type is often called a "Liquid Locker".
+
+To achieve this, Yearn has deployed a system of smart contracts that allows users to permissionlessly max-lock their governance tokens to a central Yearn in exchange for a yLocker token (e.g., yCRV, yPRISMA, etc.) at a rate of 1:1.
 
 The benefit of doing this is to provide the end user with a fully transferrable and liquid token that can still receive a share of governance benefits like yield or voting power depending on the protocol. No longer is there a need to lock up your asset value for up to 4 full years!
 
 > **⚠️ Important**: Liquid locker tokens are not redeemable for the underlying locked tokens. This is not possible. But because they are liquid, they can be traded on decentralized exchanges, and bought and sold at market value.
+
+## yLocker Products
+
+* [yPrisma](../ylockers/yprisma/overview)
+* [yCRV](../ylockers/ycrv/overview)
+
+## Earning Yield with yLockers
+
+Within the yLockers ecosystem, there are two distinct ways to earn yield:
+
+1. Deposit your yLocker tokens into YBS (earn stablecoins)
+2. Deposit your yLocker tokens into the V3 Auto-Compounder (earn yLocker tokens)
+
+___
+
+### Option 1: Deposit yLocker Tokens into YBS
+
+Each week, Yearn's yLockers earn revenue from protocol fees and bribes. This revenue is converted into ecosystem stablecoins (mkUSD for yPrisma or crvUSD for yCRV) and distributed to yLocker stakers at the start of the week via the `YearnBoostedStaker`(YBS), a new contract which was launched in Spring 2024 following the passing of [YIP-77](https://snapshot.org/#/veyfi.eth/proposal/0xe79fb2ef4f21ef1e9cc30dd1522c9751c74b631c4782bccbbeb25185d4ddae1d).
+
+To begin earning your share, all you need to do is stake your respective yLocker tokens in the staking contract. Take a look at the interface guides for your desired yLocker to get started. Want to learn how it all works under the hood? Read on Anon!
+
+### Weights and Boosts
+
+The longer you stake, the greater your boost! Yearn's yLocker staking contract incentivizes long-term users by boosting their yield (up to a maximum of 2.5x).
+
+You'll reach max boost and achieve the maximum staking APR less than four weeks after depositing your yLocker tokens.
+
+To calculate your boost, the staking contract maintains a weight for every deposit (which is a function of the amount of yLocker tokens you have staked and the duration since it was staked).
+
+Your stake weight starts at 50 points with your initial deposit, and increases by 50 points each week (at 00:00:00 UTC, Thursday) until the maximum weight is reached after 4 weeks.
```

Comment:
We actually should add/update the description here to a last-minute change that was implemented for security reasons (to avoid users being able to take yield from depositing right before epoch flip, then swap to cvxPRISMA, collect nearly a week of yield, switch back, etc.)

"For security purposes, the rewards contract is designed to ignore staked amounts in their first week. I.e. Amounts that that have not yet reached a level of 1x boost."

Reply:
So the deposit starts out unweighted, and then after the first week hits 1x boost and increases by 50 points/week from there?

Reply:
So this is where it's even more confusing (yay). The user's weight actually stays the same (0.5x in the first week) but they just don't earn any rewards during the first week

the launch announcement has a bit more details on it, and wavey and I can explain in more detail in tg as well: https://blog.yearn.fi/ybs-yprisma-launch

Reply:
let me know what you think of the new wording

> Stake weight increases by 50 points each week (at 00:00:00 UTC, Thursday) until the maximum weight is reached after 4 weeks. For security purposes, the rewards contract is designed to ignore staked amounts that have not yet reached a level of 1x boost (those in their first week). After the first week and once the stake weight has reached 100 points, rewards become claimable.